test/CppTests/TestConfigurations/config_msys_gdb.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<TestMachineConfiguration>
+  <TestConfigurations>
+    <TestConfiguration
+      Compiler="GppMingw64"
+      DebuggeeArchitecture="x64"
+      Debugger="GdbMingw64" />
+  </TestConfigurations>
+  <Compilers>
+    <Compiler
+      Name="GppMingw64"
+      Type="GPlusPlus"
+      Path="D:\a\_temp\msys\msys64\mingw64\bin\g++.exe"/>
```

Comment:
Is it okay that these are hard coded?

Reply:
> Is there something we can do to suppress the notification for all the compiler warnings? Or do we just need to prioritize disabling or fixing them?

Working on resolving the warnings right now. 

> Is it okay that these are hard coded?

I think for now that is ok. I want to have at least Windows/Linux tests running so we can review other Contributors PR's.

I also want to enable Cygwin/MinGW as separate tasks along side MSYS.
- https://github.com/marketplace/actions/install-cygwin
- https://github.com/marketplace/actions/install-mingw

Reply:
Ideally we run in msys `cygpath -w $(which [g++|gdb])` for that transform.